File: src/main/java/bearmug/lambda/PingController.java
package bearmug.lambda;

import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;
import reactor.core.publisher.Mono;

import javax.inject.Inject;

@Controller("/ping")
public class PingController {

	@Inject
	PingService pingService;

    @Get("/java")
    public Mono<String> pingJava() {
        return Mono.just(pingService.pong());
    }

    @Get("/graal")
    public Mono<String> pingGraal() {
        return Mono.just(pingService.pong());
    }
}

File: src/main/kotlin/bearmug/lambda/PingController.kt
package bearmug.lambda

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get
import reactor.core.publisher.Mono

@Controller("/ping")
class PingController(
        private val pingService: PingService) {

    @Get("/java")
    fun pingJava() = Mono.just(pingService.pong())

    @Get("/graal")
    fun pingGraal() = Mono.just(pingService.pong())
}